MySql/ConfigurationDbContext.sql

```diff
@@ -162,9 +162,9 @@ FROM `IdentityProperties`;
 
 -- ApiScopes -> ApiResourceScopes
 INSERT INTO `ApiResourceScopes` 
- (`Scope`, `ApiResourceId`)
+ (`Id`, `Scope`, `ApiResourceId`)
 SELECT 
- `Name`, `ApiResourceId`
+ `Id`, `Name`, `ApiResourceId`
```

Comment:
**question** Do we still need to include the `Id` field in the query if we're now using AUTO_INCREMENT in the table creation script?